### PR TITLE
Removed CCRI example from FAQ to avoid confusion.

### PR DESCRIPTION
**Pull request by pphector**
We need to remove Guillaume's CCRI ID from this page because we had an external user accidentally request sponsorship from our group because they got confused by this section. I edited the source file with a fix.

### CODE REVIEW

File: docs/source/faq/faq_new_users.rst [hunk 1]
```diff
@@ -27,17 +27,15 @@ To create a `new CCDB account`_, what should I fill in the form field: 'position
 
 **Response**
 
-Use the following option in the form:
+Chose the appropriate option in the form, for example:
```
Comment (shaloo) [inline docs/source/faq/faq_new_users.rst]:
```suggestion
Choose an appropriate option in the form, for example:

**Review by shaloo — APPROVED** [review]
Fixed text and typos. Approving this change

**Comment by shaloo** [comment]
Merging to main and publishing.

**Comment by pphector** [comment]
Thanks Shaloo!

**Pull request merged**